jaeger_client/span.py

```diff
@@ -182,3 +187,17 @@ def error(self, message, payload=None):
         else:
             self.log(event=message)
         return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
this is already defined on the OpenTracing span that we extend. Why do we need to override it instead of doing putting this logic into OT?